src/tutorial/runtime/introduction.rst

```diff
@@ -276,7 +260,7 @@ Where Do All The Files Go?
       :term:`job log files <job log>` by right-clicking on the task and
       selecting "View".
 
-      .. image:: ../img/cylc-gui-view-log.png
+      .. TODO REPLACE THIS IF APPROPRIATE: .. image:: ../img/cylc-gui-view-log.png
```

Comment:
This content is not valid at the moment, will need to be commented out.

Can replace with the `cylc cat-log` command for the time being.

Reply:
This one makes sense, as the rest of the tip needs commenting and replacing with the cylc cat-log comment.